### PR TITLE
First draft for review

### PR DESCRIPTION
**Pull request by artpropp**
This is a project management guidelines draft for development here on GitHub. Please feel free to amend, comment, extend or shorten where needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
 # OrgApp development guidelines
 Dev conventions and guidelines for the OrgApp project
+
+------
+Basic project workflow will apply `Projects` as on `GitHub`.
+Within a repository one will find the `Projects` tab, right between `Pull requests` and `Wiki`. The `GitHub` `Projects` will be used like a kind of **scrum board** with a _backlog_ column, a _sprint backlog_, a _in progress_, a _in review_, and a _closed_ or _done_ column.
```
Comment (DennisAhaus) [inline README.md]: Please specify concrete - not `closed` or `done` but one of this

File: README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
...
+
+Once a note within the project _backlog_ is moved from the _backlog_ to a _sprint backlog_ this note should be made an issue. There is a handy feature for that, and it lives inside the `GitHub` `Projects` notes. Simply click on flipped caret symbol (`∨`) in the right corner of a note, and then click on `Convert to issue`.
+Once you converted that note into an issue open the `Issues` tab within the current repository and edit that issue you created by converting it from that note.
+
+Now you can assign this issue to a milestone, add labels to it, and even set an assignee for this very issue. But make sure that once an issue has been assigned to you or another developer within the team this issue has also to be moved from the _sprint backlog_ column to the _in progress_ column within the `Projects` tab. By this means a _sprint_ will get a structured backlog and visible progress, all within the `GitHub` `Projects`.
```
Comment (DennisAhaus) [inline README.md]: You `can` or you have to i.e. `assign` ?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,28 @@
...
+ 
+We use `GitHub` _milestones_ as _sprints_, which also have a start and an end date. And simply think of the _sprint backlog_ as of issues within a milestone that have not been assigned. Once assigned, those items have to be moved to _in progress_ column.
+
+The table below illustrates an example scrum board with its corresponding columns and their  transfer to the `GitHub` features.
+
+|       Backlog     | Sprint Backlog |   In Progress  |        In Review       |                       Closed                    |
+|:-----------------:|:--------------:|:--------------:|:----------------------:|:-----------------------------------------------:|
+| without milestone | with milestone | with milestone |      with milestone    | contributes progress to corresponding milestone |
```
Comment (DennisAhaus) [inline README.md]: I think for `In Progress` there should be an assignee available!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,28 @@
...
+|     unassigned    |   unassigned   |    assigned    |         assigned       |                                                 |
+|                   |                |                | pull request reference |                                                 |
+
+At this very moment the `GitHub` `Projects` do only stick to one repository. With this minor drawback in mind the overall global backlog might be shared and cultivated at some other place.
```
Comment (DennisAhaus) [inline README.md]: What we do here? Please make a concrete suggestion and don't use words like `might` or `eventually` or whatever

**Comment by artpropp** [comment]
In consideration of @DennisAhaus' comment regarding the explanation of the `Projects` API, I changed this draft into a more generic one, which is not trying to describe the current state of the GitHub`Projects.

**Comment by DennisAhaus** [comment]
👍

**Pull request merged**